### PR TITLE
WIP: Migrating Go module to in-toto

### PR DESCRIPTION
**Pull request by ChaosInTheCRD**
This PR is to change the name of the module and all imports inside it to reflect the projects new location at github.com/in-toto/go-witness.

I've tried my best to catch every use of `testifysec/go-witness` but this will probably need some more validation / testing.

### CODE REVIEW

**Comment by ChaosInTheCRD** [comment]
This will fail for now as we are waiting for merging of https://github.com/in-toto/archivista/pull/121

File: archivista/download.go [hunk 1]
```diff
@@ -17,8 +17,8 @@ package archivista
 import (
 	"context"
 
-	archivistaapi "github.com/testifysec/archivista-api"
-	"github.com/testifysec/go-witness/dsse"
+	archivistaapi "github.com/in-toto/archivista/pkg/api"
```
Comment (jkjell) [inline archivista/download.go:20]: Let's move to separate PR

File: archivista/searchgitoids.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ package archivista
 import (
 	"context"
 
-	archivistaapi "github.com/testifysec/archivista-api"
+	archivistaapi "github.com/in-toto/archivista/pkg/api"
```
Comment (jkjell) [inline archivista/searchgitoids.go:20]: move to separate PR

File: archivista/store.go [hunk 3]
```diff
@@ -17,8 +17,8 @@ package archivista
 import (
 	"context"
 
-	archivistaapi "github.com/testifysec/archivista-api"
-	"github.com/testifysec/go-witness/dsse"
+	archivistaapi "github.com/in-toto/archivista/pkg/api"
```
Comment (jkjell) [inline archivista/store.go:20]: move to separate PR

**Review by jkjell — CHANGES_REQUESTED** [review]
I think we remove the archivista package changes we can hopefully at least get tests running.

**Comment by ChaosInTheCRD** [comment]
So CI is going to continue failing on this PR as the old `archivista-api` module refers to the old `go-witness` module. I suppose this will just wait until https://github.com/in-toto/archivista/pull/121 is merged?

**Review by jkjell — APPROVED** [review]

**Review by jkjell — APPROVED** [review]

**Comment by ChaosInTheCRD** [comment]
@jkjell just to confirm, the current state is that we are depending on the main branch of https://github.com/in-toto/archivista and we will remove this dependency back to the next release when that is cut?

**Pull request merged**